addons/binding/org.openhab.binding.amazonechocontrol/README.md

```diff
@@ -0,0 +1,303 @@
+
+## Warning ##
+
+For the connection to the Amazon server, your password of the Amazon account is required, this will be stored in your openHAB thing device configuration. So you should be sure, that nobody other has access to your configuration! 
+
+## What else you should know ##
+
+All the display options are updated by polling the amazon server. The polling time can be configured, but a minimum of 10 seconds is required. The default is 60 seconds, which means the it can take up to 60 seconds to see the correct state. I do not know, if there is a limit implemented in the amazon server if the polling is too fast and maybe amazon will lock your account. 60 seconds seems to be safe.
+
+## Supported Things
+
+| Thing type id        | Name                                  |
+|----------------------|---------------------------------------|
+| account              | Amazon Account                        |
+| echo                 | Amazon Echo Device                    |
+| echospot             | Amazon Echo Spot Device               |
+| echoshow             | Amazon Echo Show Device               |
+| wha                  | Amazon Echo Whole House Audio Control |
+| flashbriefingprofile | Flash briefing profile                |
+| unknown              | Unknown Echo Device or App\*          |
+
+\* The unknown device will provide all channels, but maybe not all of them supported by your device.
+
+## Discovery
+
+The first 'Amazon Account' thing will be automatically discovered. After configuration of the thing with the account data, a 'Amazon <???>' thing will be discovered for each registered device. If the device type is not known by the binding, an 'Unknown' device will be created.
+
+## Binding Configuration
+
+The binding does not have any configuration. The configuration of your amazon account must be done in the 'Amazon Account' device.
+
+## Thing Configuration
+
+The Amazon Account thing need the following configurations:
+
+| Configuration name       | Description                                                               |
+|--------------------------|---------------------------------------------------------------------------|
+| amazonSite               | The amazon site where the echos are registered. e.g. amazon.de            |
+| email                    | Email of your amazon account                                              |
+| password                 | Password of your amazon account                                           |
+| pollingIntervalInSeconds | Polling interval for the device state in seconds. Default 60, minimum 10  |
+
+2 factor authentication is not supported!
+
+** HINT ** IMPORTANT: If the Account thing does not go online and reports a login error, open the url YOUR_OPENHAP/amazonechocontrol/ID_OF_ACCOUNT_THING (Replace YOUR_OPENHAB and ID_OF_ACCOUNT_THING with your configuration) in your browser (e.g. http://openhab:8080/amazonechocontrol/account) and try to login.
```

Comment:
Yes for captchas and sometime amazon requires to confirm a code sent to the mobile phone. But fortunately this seem only be happen for the first login from the openHAB installation, so the session renewal can be done automatically. And I think about the option, to remove the configuration of the account data from the account thing, because than I could store the password encrypted. I think it is a little bit danger to have such information in clear text in the configuration (Of course I know, the binding need the encryption key and a hacker can get them from the source code, but it would be a extra step). But maybe open HAB itself should provide this feature, if a configuration is attributed as password ;)